separate_instance/database/src/config/index.ts

```diff
@@ -0,0 +1,40 @@
+// ATTENTION: DO NOT PUT ANY SECRETS IN HERE (or the .env)
```

Comment:
whats the change in this file?

Reply:
...
  DB_PORT: process.env.DB_PORT ? parseInt(process.env.DB_PORT) : 3307,
...